build-tests/rush-redis-cobuild-plugin-integration-test/sandbox/sharded-repo/rush.json

```diff
@@ -0,0 +1,41 @@
+{
+  "rushVersion": "5.80.0",
```

Comment:
This is a pretty old version.

Reply:
I copied the existing cobuild sandbox repo for this. It uses the locally installed version of rush, though the start-dev script and doesn't use this version AFAIK.

Reply:
Fair enough.